src/rgw/rgw_json_enc.cc

```diff
@@ -904,6 +904,7 @@ void RGWZoneParams::dump(Formatter *f) const
   encode_json("log_pool", log_pool, f);
   encode_json("intent_log_pool", intent_log_pool, f);
   encode_json("usage_log_pool", usage_log_pool, f);
+  encode_json("roles_pool", roles_pool, f);
```

Comment:
Would be good to write why roles_pool is added or create an tracker & link here for future reference.

Reply:
just provide a way to set roles_pool